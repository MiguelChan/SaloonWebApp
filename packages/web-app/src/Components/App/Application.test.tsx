import React from 'react';
import {
  render,
  RenderResult,
} from '@testing-library/react';
import { Application } from '.';

describe('Application', () => {
  it('Should render the Application', () => {
    const renderResult = setupComponent();
    expect(renderResult.container).toMatchSnapshot();
  });

  const setupComponent = (): RenderResult => {
    return render(<Application />);
  };

});