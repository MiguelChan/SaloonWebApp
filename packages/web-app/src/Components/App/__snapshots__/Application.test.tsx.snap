// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application Should render the Application 1`] = `
<div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-1ps6pg7-MuiPaper-root"
  >
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-bhp9pd-MuiPaper-root-MuiCard-root"
    >
      <div
        class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-16m3jxf-MuiTypography-root"
        >
          Trabajando arduamente
        </p>
        <div
          class="MuiTypography-root MuiTypography-h5 css-ag7rrr-MuiTypography-root"
        >
          Estamos actualmente trabajando en hacer esta pagina
        </div>
      </div>
    </div>
  </div>
</div>
`;
